src/Twig/Handler/ImageHandler.php

```diff
@@ -155,12 +155,20 @@ public function showImage($fileName = null, $width = null, $height = null, $crop
         }
         $thumb = $this->getThumbnail($fileName, $width, $height, $crop);
 
-        if ($width === null || $height === null) {
+        if ($width === null && $height === null) {
+            $thumbconf = $this->app['config']->get('general/thumbnails');
+            $width = !empty($thumbconf['default_image'][0]) ? $thumbconf['default_image'][0] : 1000;
+            $height = !empty($thumbconf['default_image'][1]) ? $thumbconf['default_image'][1] : 750;
+            $thumb->setWidth($width);
+            $thumb->setHeight($height);
```

Comment:
I might be missing something here, but isn't this already done in `\Bolt\Helpers\Image\Thumbnail`? 

The point of that class was to abstract, and centralise all of this :wink: